scripts/update_copyright_date.sh

```diff
@@ -44,7 +44,7 @@
 #=============================================================================
 # First find all the files we want to modify
 #=============================================================================
-find . -type f ! -name \*.git\* ! -name \*update_copyright\* -exec grep -l "other Axom Project Developers. See the top-level COPYRIGHT file for details." {} \; > files2change
+git grep -l "other Axom Project Developers. See the top-level COPYRIGHT file for details." | grep -v update_copyright > files2change
```

Comment:
@ltaylor16 -- The old way was finding some files in ``.git`` and in build directories.  This new command is significantly faster and finds the correct set of files